### DISCUSSION TITLE
【android】求助！编译android的库，报错'strip' 不是内部或外部命令，也不是可运行的程序

### DISCUSSION THREAD

**Discussion by xzpqjew** [Tencent/tgfx#43] [Q&A] [answered]
![企业微信截图_17006362117704](https://github.com/Tencent/tgfx/assets/37336674/471eec2d-b6a4-480b-a728-ad01d5912f42)
问题报错如上

**Comment by wuqi** [comment]
strip是linux的命令,你这为啥不提issue

**Reply by xzpqjew** [reply]
感谢回复，最后在电脑上找到个strip.exe，把这个路径设置到环境变量里，就可以解决这错误，但后面又遇到了js 报错system call “mkdir”命令不识别，看来这个编译脚本在windows电脑上编android平台的库，根本用不了

**Reply by wuqi** [reply]
你要不试试在msys2或者wsl下编

**Reply by domchen** [reply, marked as answer]
这个是因为之前我们并不支持在Windows上编译Android，报错具体问题是编译系统找不到NDK，因为团队日常开发的设备是mac，可以一次性覆盖除了Windows外几乎所有平台。现在更新到main分支最新代码已经可以支持了在Windows 编译Android了。另外NDK的版本也扩展到大于19的都可以支持了。
